tools/sqlite3_api_wrapper/sqlite3_api_wrapper.cpp

```diff
@@ -391,6 +391,9 @@ int sqlite3_column_type(sqlite3_stmt *pStmt, int iCol) {
 	case LogicalTypeId::DATE:
 	case LogicalTypeId::TIME:
 	case LogicalTypeId::TIMESTAMP:
+	case LogicalTypeId::TIMESTAMP_SEC:
```

Comment:
Could you add some C API tests involving these types as well (test/api/capi/test_capi.cpp)?